### ISSUE TITLE
MSVC-Debug too slow

### ISSUE THREAD

**Issue by jalegido** [FreeTDS/freetds#448]
When compiling in Debug with MSVC, freetds is changing some MSVC internal debug variables that affect the performance of the whole application.
All messages reported to output debug window are shown in stdout. This makes a poor performance in the application if it use intensively this debug messages.

```
#if defined(_MSC_VER) && defined(_DEBUG)
		_CrtSetReportMode(_CRT_WARN, _CRTDBG_MODE_FILE);
		_CrtSetReportFile(_CRT_WARN, _CRTDBG_FILE_STDOUT);
		_CrtSetReportMode(_CRT_ERROR, _CRTDBG_MODE_FILE);
		_CrtSetReportFile(_CRT_ERROR, _CRTDBG_FILE_STDOUT);
		_CrtSetReportMode(_CRT_ASSERT, _CRTDBG_MODE_FILE);
		_CrtSetReportFile(_CRT_ASSERT, _CRTDBG_FILE_STDOUT);
		_CrtSetDbgFlag(_CRTDBG_CHECK_ALWAYS_DF | _CRTDBG_LEAK_CHECK_DF | _CrtSetDbgFlag(_CRTDBG_REPORT_FLAG));
#endif
```

Does any modification exist to solve this performance problem?

Best regards

**Comment by fziglio** [comment]
There's no performance issue. If you enable debug don't use in production.

**Comment by jalegido** [comment]
Yes, I know that in production must not use this Debug code.
The problem appears while develop, the whole application is slow; because all traces go to stdout instead only msvc output window.

**Comment by fziglio** [comment]
What's the actual situation? Unless you are debugging FreeTDS code that's no need to use a debug version of FreeTDS.
Having output on stdout could be an issue. Not clear why not using stderr which seems more appropriate.

**Comment by jalegido** [comment]
Yes, you are right that we should use release version of FreeTDS when we develop. 
But, we are using vcpkg as package manager and it compile FreeTDS in debug mode and forces to link with this version while we are developing.
When we produce release images, vcpkg use the release version of FreeTDS and problem disappears.
